static/scss/product-details.scss

```diff
@@ -0,0 +1,260 @@
+    padding: 38px 0;
+  }
+
+  .content-row {
+    @media (min-width: 1024px) {
+      display: flex;
+      flex-flow: row wrap;
+      margin: 0 -15px;
+    }
+
+    .content-col {
+      padding: 0 0 20px;
+      @media (min-width: 768px) {
+        padding: 0 0 30px;
+      }
+      @media (min-width: 1024px) {
+        width: calc(100% - 455px);
+        padding: 0 15px;
+        box-sizing: border-box;
+      }
+
+      &.video {
+        padding-bottom: 0;
+        @media (min-width: 1024px) {
+          width: 455px;
+          padding: 0 15px;
+          flex-shrink: 0;
+        }
+      }
+    }
+  }
+
+  .video-holder {
+    background: $sirocco;
+    margin-top: 10px;
+
+    img {
+      display: block;
+      width: 100%;
+    }
+  }
+
+  .text-info {
+    @media (min-width: 1024px) {
+      max-width: 625px;
+    }
+  }
+
+  h1 {
+    font-size: 42px;
+    line-height: 50px;
+    margin: 0 0 13px;
+    font-weight: 700;
+
+    @media (min-width: 768px) {
+      font-size: 48px;
+      line-height: 60px;
+    }
+  }
+
+  p {
+    margin: 0 0 28px;
+  }
+
+  .btn.btn-default {
+    font-size: 20px;
+    line-height: 30px;
+    color: #fff;
+    min-height: 60px;
+    border: 0;
+    box-sizing: border-box;
+    padding: 15px 20px;
+    text-align: center;
+    min-width: 230px;
+    display: inline-block;
+    vertical-align: top;
+    background: linear-gradient(174.8deg, #c5264e 0%, #951328 100%);
+    text-decoration: none;
+    font-weight: 700;
+    transition: linear 0.4s;
+
+    &:hover {
+      opacity: 0.8;
+    }
+  }
+}
+
+.info-block {
+  font-family: 'Montserrat', sans-serif;
+  padding: 20px 0;
+  font-size: 14px;
+  line-height: 16px;
+  color: $black;
+
+  @media (min-width: 1024px) {
+    padding: 30px 0;
+  }
+
+  .stats-row {
+    display: flex;
+    flex-flow: row wrap;
+    margin: 0 -8px;
+    align-items: baseline;
+
+    @media (min-width: 1200px) {
+      margin: 0 -18px;
+    }
+
+    .stat-col {
+      margin: 0 8px 16px;
+      background: $very-light-gray;
+      border-radius: 3px;
+      overflow: hidden;
+      text-align: center;
+      display: flex;
+      flex-flow: column;
+      flex-grow: 1;
+      width: calc(50% - 16px);
+
+      @media (min-width: 1024px) {
+        width: auto;
+      }
+
+      @media (min-width: 1200px) {
+        margin: 0 18px;
+        width: 180px;
+        flex-grow: inherit;
+      }
+
+      &.small {
+        @media (min-width: 1200px) {
+          width: 140px;
+        }
+      }
+
+      &.large {
+        flex-grow: 1;
+        text-align: left;
+        }
+      }
+    }
+
+  .text {
+    padding: 14px 20px;
+    width: 100%;
+    flex-grow: 1;
+    display: inline-block;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    font-size: 14px;
+
+    p {
+      margin: 0;
+    }
+
+    a:link {
+      color: $link-blue;
+      text-decoration: underline;
+    }
+  }
+
+  .title {
+    display: block;
```

Comment:
If we're using `display: block` and setting width to 100%, any reason we wouldn't just use a `div` instead of a `span`?